vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini_authorization.cc

```diff
@@ -151,6 +150,12 @@ void GeminiAuthorization::OnFetchRecipientId(
     const std::string& recipient_id,
     const std::string& token,
     ledger::ExternalWalletAuthorizationCallback callback) {
+  if (result == type::Result::NOT_FOUND) {
```

Comment:
It's not quite ideal to use `NOT_FOUND` here, but if it's the best `ledger::Result` available then it's fine.

Reply:
@zenparsing  - we've used `NOT_FOUND` for uphold as well, so using the same for now. I think we should be using `REGISTRATION_VERIFICATION_FAILED` instead, I think doing this as a follow-up change makes sense, let me know if you disagree.

Reply:
Sounds good.